services/galaxy-toolshed/galaxy-toolshed-activity.service.js

```diff
@@ -0,0 +1,70 @@
+import Joi from 'joi'
+import { formatDate } from '../text-formatters.js'
+import { age as ageColor } from '../color-formatters.js'
+import BaseGalaxyToolshedService from './galaxy-toolshed-base.js'
+
+const repositoryRevisionInstallInfoSchema = Joi.array().items(
+  Joi.object({
+    create_time: Joi.string().required(),
+  }).required(),
+  Joi.object({}),
+  Joi.object({})
+)
+
+export default class GalaxyToolshedActivity extends BaseGalaxyToolshedService {
+  static category = 'activity'
+  static route = {
+    base: 'galaxy-toolshed/create-date',
+    pattern: ':repositoryName/:owner',
+  }
+
+  static examples = [
+    {
+      title: 'Galaxy Toolshed - Create Time',
```

Comment:
Similarly here, we've stuck with the response field name of `Time` even though we're displaying a date. This should be updated to be consistent with whatever term we go with, see below comment

Reply:
You are right, I will update the code